projects/experimental/components/sensitive/sensitive.style.less

```diff
@@ -0,0 +1,21 @@
+@import 'taiga-ui-local';
+
+.tui-sensitive {
+    position: relative;
+    display: inline-block;
+    line-height: 1em;
+
+    &:before {
+        .fullsize();
+        content: '';
+        background: currentColor;
+        border-radius: inherit;
```

Comment:
Should we have some default border-radius on the host?

Reply:
let's make 0.25rem by default